### ISSUE TITLE
Cran update

### ISSUE THREAD

**Issue by mvfki** [zdebruine/RcppML#44]
Hi RcppML team,

It seems that the GitHub dev version has already reached 0.5.x while the version on cran is still 0.3.x. I'm working on a package and would love to include your methods as a dependency. While the cran version is really out-of-date (I can see that those R version-related errors are already fixed in the dev version), I would like to request a cran update with your latest stable version. 

Sincerely,
Yichen

**Comment by zdebruine** [comment]
Yes, I will be working on this through June and expect a stable v1.0 on CRAN by the end of June.  Thanks for your interest.

**Comment by Yunuuuu** [comment]
Any updates for this?
